Error occured at publishing modules

Run yarn config set -H 'npmAuthToken' "***"
➤ YN0000: Successfully set npmAuthToken to '********'
Usage Error: Invalid option schema: missing at least one property from "all", "recursive", "since", or "worktree"

$ yarn workspaces foreach [--from #0] [-A,--all] [-R,--recursive] [-W,--worktree] [-v,--verbose] [-p,--parallel] [-i,--interlaced] [-j,--jobs #0] [-t,--topological] [--topological-dev] [--include #0] [--exclude #0] [--no-private] [--since] [-n,--dry-run] <commandName> ...
Error: Process completed with exit code 1.

--- a/.github/workflows/publish.yaml
+++ b/.github/workflows/publish.yaml
@@ -96,7 +96,7 @@ jobs:
       - name: publish
         run: |
           yarn config set -H 'npmAuthToken' "${{secrets.NPM_TOKEN}}"
-          yarn workspaces foreach -p -j 10 -v --no-private npm publish --access public --tolerate-republish
+          yarn workspaces foreach --all -p -j 10 -v --no-private npm publish --access public --tolerate-republish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
       # - name: publish

